### PR TITLE
Add support for multi buildpack

### PR DESCRIPTION
**Pull request by dgodd**
Thanks for contributing to the buildpack. To speed up the process of reviewing your pull request please provide us with:

* A short explanation of the proposed change:
Convert to new supply/finalize format for multi buildpack support

* An explanation of the use cases your change solves
Support multi buildpacks

* [ x] I have viewed signed and have submitted the Contributor License Agreement

* [ x] I have made this pull request to the `develop` branch

* [x ] I have added an integration test
No perecived changes for users unless using multi buildpacks

### CODE REVIEW

**Comment by sclevine** [comment]
@opiethehokie seems like @degrim is unavailable? Mind reviewing this PR?

**Comment by opiethehokie** [comment]
@sclevine I'm out of the office this week. If @degrim can't get to it sooner I'll be able to review Tuesday. Sorry for the delay.

File: cf_spec/spec_helper.rb [hunk 1]
```diff
@@ -12,9 +12,9 @@
 RSpec.configure do |config|
   config.color = true
   config.tty = true
-  config.verbose_retry = true
-  config.default_retry_count = 3
-  config.default_sleep_interval = 5
+  # config.verbose_retry = true
```
Comment (degrim) [inline cf_spec/spec_helper.rb]: Is it intended to remove the retries here?

Comment (dgodd) [inline cf_spec/spec_helper.rb]: No it wasn't. Thanks I've put them back

**Pull request merged**